packages/integrations/useFocusTrap/index.ts

```diff
@@ -86,13 +87,16 @@ export function useFocusTrap(
     }
   }
 
+  const targets = computed(() => Array.isArray(target)
```

Comment:
We need to consider that `target` might be a `Ref<HTMLElement[]>`.

Reply:
cc: [multiple target](https://github.com/vueuse/vueuse/blob/main/packages/core/useIntersectionObserver/index.ts)

I think it should be the same as `useIntersectionObserver`

Reply:
Updated, thanks for the tips!